tests/functional/docs/test_smoke.py

```diff
@@ -19,6 +19,12 @@
 from boto3.docs.service import ServiceDocumenter
 
 
+# This is a temporary hack
+def _snake2camel(s):
```

Comment:
What was the reason for adding this as opposed to what I did in my pending PR? I know you labeled it as temporary, but it seems super fragile.